### DISCUSSION TITLE
LaTeX-style template?

### DISCUSSION THREAD

**Discussion by user202729** [typst/typst#517] [Q&A] [answered]
Is there a template that mimics LaTeX default `article` class output?

Here is my attempt:

```
#let conf(title: none, author: none, doc) = [
  #set text(font: "Latin Modern Roman", size: 10pt)
  #set par(first-line-indent: 20pt, justify: true)
  #set page(
    margin: (
      left: 4.5cm,
      right: 4.5cm,
      top: 4.5cm,
      bottom: 4.5cm,
    ),
    numbering: "1",
  )

  #v(2cm)
  #align(center)[
    #set par(first-line-indent: 0pt)
    #text(17pt, title)

    #text(12pt)[#author

    April 1, 2023
  ]
  #doc
]

/* use like this
#import "Typst-LaTeX-template.typ": conf
#show: doc => conf(
  title: [title],
  author: [author],
  doc)
  */
```

A few things are off though. (e.g. normally LaTeX does not indent the very first paragraph, or paragraphs inside itemize),
and interparagraph spacing)

**Comment by user202729** [comment, marked as answer]
Never mind, the ams template pretty much solve it.

(the template is on the website, or can be downloaded from Typst's Discord server https://discord.com/channels/1054443721975922748/1088501462910378015/1088508263001227386)

**Reply by lvjr** [reply]
It is in https://github.com/typst/templates.
